app/views/planning_applications/steps/_assessment.html.erb

```diff
@@ -1,5 +1,14 @@
 <h1 class="govuk-heading-l">Assessment</h1>
 <ul class="app-task-list__items" id="assess-section">
+  <li class="app-task-list__item">
+    <span class="app-task-list__task-name">
+      <% if @planning_application.validated? %>
+        <%= link_to "Review non-validation requests", post_validation_requests_planning_application_validation_requests_path(@planning_application), class: "govuk-link" %>
+      <% else %>
+        Review non-validation requests
+      <% end %>
```

Comment:
Is this a case for `#link_to_if`? https://apidock.com/rails/v5.2.3/ActionView/Helpers/UrlHelper/link_to_if